ui/app/pages/routes/index.js

```diff
@@ -98,6 +98,27 @@ class Routes extends Component {
     })
   }
 
+  componentDidUpdate (prevProps) {
+    const {
+      threeBoxSynced,
+      show3BoxRestoreConfirmModal,
+      getThreeBoxLastUpdated,
+      setThreeBoxSyncingPermission,
+      restoredFromThreeBox,
+    } = this.props
+
+    if (!prevProps.threeBoxSynced && threeBoxSynced && restoredFromThreeBox === null) {
+      getThreeBoxLastUpdated()
+        .then(lastUpdated => {
+          if (lastUpdated) {
+            show3BoxRestoreConfirmModal(lastUpdated)
+          } else {
+            setThreeBoxSyncingPermission(true)
```

Comment:
Similar to the other case, I don't see how this could reach this state without this permission being `true` already.

Reply:
You are right again, and same as before this needs to be replaced with `turnThreeBoxSyncingOn()`